iis-common/src/main/java/eu/dnetlib/iis/common/model/extrainfo/ExtraInfoConstants.java

```diff
@@ -11,9 +11,9 @@ public final class ExtraInfoConstants {
 	public static final String NAME_RESULT_STATISTICS = "result statistics";
 	public static final String NAME_AUTHOR_STATISTICS = "author statistics";
 	
-	public static final String NAME_CITATIONS = "result citations";
+	public static final String NAME_REFERENCES = "result references";
 	
-	public static final String TYPOLOGY_CITATIONS = "citations";
+	public static final String TYPOLOGY_REFERENCES = "references";
```

Comment:
I get that this is what was specified (as much as anything was) in the original redmine ticket. But I wonder - having no actual idea - is this typology certain not to clash with something else that we also call references? Maybe it should be bibliographic-references or something similar to be more specific?

Reply:
I agree, this i a good point, I have created a note on that matter on redmine: https://support.openaire.eu/issues/7509#note-16.